### PR TITLE
Docs: Add Themes page in Zh-TW

### PR DESCRIPTION
**Pull request by ymcheung**
## Changes
Themes page in zh-TW

## Testing
There should be a `佈景主題` link at left side bar.

## Docs
This is a doc update.

### CODE REVIEW

**Comment by matthewp** [comment]
Hey @ymcheung, thanks for submitting this! We are just waiting on a review from a Zh-TW speaker (have reached out on Discord).

**Review by drwpow — APPROVED** [review]
Thank you!

**Comment by drwpow** [comment]
Ah I did not see Matthew’s comment. Will await another review before merging, then.

**Review by flikteoh — COMMENTED** [review]
Just a bit of wording suggestion to be a little more specific about community packages/plugins.

Not quite sure `主打` is the right word?

File: docs/src/pages/zh-TW/themes.astro
Comment (ymcheung) [reply docs/src/pages/zh-TW/themes.astro:43]: `精選` is more accurate than `主打`, thank you!

**Comment by ymcheung** [comment]
Updated! :)

**Comment by jonathantneal** [comment]
@flikteoh do you think the updates make this PR ready to merge?

**Comment by flikteoh** [comment]
@jonathantneal Yeah it's looking all good! Thank you @ymcheung

**Comment by natemoo-re** [comment]
I think this just needs to be rebased against `main` and everything should pass again! Thanks for the PR!

**Comment by ymcheung** [comment]
> I think this just needs to be rebased against `main` and everything should pass again! Thanks for the PR!

Finally I understood why it failed all the time. Fixed!

**Comment by natemoo-re** [comment]
Hooray, thank you so much @ymcheung! Merging this now.

**Comment by flikteoh** [comment]
Layout seems to have broken. [https://docs.astro.build/zh-tw/themes/](https://docs.astro.build/zh-tw/themes/) 😅

**Comment by FredKSchott** [comment]
Reverted b098f04dc3fa927d9ac6c13e979ca88c1ecff5ef

This appeared to never work based on the preview URL https://deploy-preview-1725--astro-docs-2.netlify.app/zh-tw/themes/. If this worked in dev but not in the final build, let us know! This could be an Astro bug!

**Pull request merged**
